pygmt/datasets/load_remote_dataset.py

```diff
@@ -226,6 +226,29 @@ class GMTRemoteDataset(NamedTuple):
             "03m": Resolution("03m", registrations=["gridline"], tiled=True),
         },
     ),
+    "mars_relief": GMTRemoteDataset(
+        title="Mars relief",
+        name="mars relief",
+        long_name="Mars relief",
```

Comment:
Please see https://www.generic-mapping-tools.org/remote-datasets/mars-relief.html.

```suggestion
        title="Mars relief",
        name="mars relief",
        long_name="NASA Mars (MOLA) relief",
```